style.css

```diff
@@ -258,25 +261,162 @@ body.lock {
     width: 40px;
     background-color: var(--main-bg-color);  
     border: 2px solid var(--main-border-color);  ;
-    font-size: 20px;
     right: 25px;
 }
-.arrow_left {
+.arrow-left {
     border: solid var(--arrow-color);
     border-width: 0 2px 2px 0;
     display: inline-block;
     padding: 4px;
     transform: rotate(135deg);
 }
-.arrow_right {
+.arrow-right {
     border: solid var(--arrow-color);
     border-width: 0 2px 2px 0;
     display: inline-block;
     padding: 4px;
     transform: rotate(-45deg);
 }
 .content {
-    padding: 70px 0 0;
+    margin: 60px 0 0;
+}
+.carousel {
+    position: relative;
+    width: 100vw;
+    height: 140vw;
+    overflow: hidden;
+}
+.carousel__text {
+    position: absolute;
+    top: 45%;
+    left: 8%;
+    text-align: left;
+    display: active;
+    z-index: 20;
+}
+.carousel__title {
+    color: var(--light-gray-color);
+    font-size: 27px;
+    letter-spacing: 3px;
+}
+.carousel__subtitle {
+    color: var(--brown-color);
+    font-size: 22px;
+    letter-spacing: 2px;
+}
+.carousel__box {
+    position: absolute;
+    width: 100vw;
+}
+.carousel__box.middle {
+    transition: 0.5s;
+    left: 0%;
+}
+.carousel__box.left {
+    transition: 0.5s;
+    left: -100%;
+}
+.carousel__box.right {
+    transition: 0.5s;
+    left: 100%;
+}
+.carousel__object {
+    width: 100vw;
+    left: 0;
+    top: 0;
+}
+.carousel__buttons {
+    position: absolute;
+    top: 70%;
+    left: 8%;
+    z-index: 10;
+}
+.carousel__button-left, .carousel__button-right {
+    height: 60px;
+    width: 60px; 
+    border: 3px solid var(--light-gray-color);  
+    background-color: unset;
+    margin-right: 10px;
+    outline: none;
+}
+.arrow_carousel {
+    padding: 6px;
+    border: solid var(--light-gray-color);
+    border-width: 0 3px 3px 0;
+}
+.about-part {
+    position: static;
+    margin: 30px 10px; 
+    overflow: auto; 
+}
+.about-part__img-container {
+    position: relative;
+    height: 330px;
+}
+.about-part__img-container_img1 {
+    position: absolute;
+    left: 0;
+    top: 140px;
+    z-index: 10;
+    height: 150px;
+}
+.about-part__img-container_img2 {
+    position: absolute;
+    right: 0; 
+    top: 0; 
+    height: 150px;
+}
+.title {
+    font-size: 32px;
+    letter-spacing: 8px;
+}
+.subtitle {
+    color: var(--brown-color);
+    font-family: var(--main-text-font);
+    font-size: 12px;
+    letter-spacing: 1px;
+    text-transform: uppercase;
+    padding: 10px 0;
+}
+.about-part__text-container_text {
+    text-size-adjust: auto;
+    font-family: var(--main-title-font);
+    font-size: 12px; 
+    letter-spacing: 2px; 
+    line-height: 2.2;
```

Comment:
2.2 what?

Reply:
A number that will be multiplied with the current font-size to set the line height

Reply:
So how much would that be?

Reply:
2.2 * 12 = 26.4px (rounded to the side acceptable for the monitor)
If it's disturbing, can change to 26 px.

Reply:
Then why not just set 26px? Will every developer have to calculate it?

Reply:
> Then why not just set 26px? Will every developer have to calculate it?

The proportion is convenient if I want to change the font-size and then the line-height will change automatically

Reply:
Why would you change line height automatically if designer asks you to reduce font size by 2px ?

Reply:
You are setting yourself up for a bug this way.

Reply:
I already changed it to 26px